codespell_lib/data/dictionary.txt

```diff
@@ -2766,27 +2867,36 @@ devels->delves
 deveolpment->development
 devestated->devastated
 devestating->devastating
+device-dependend->device-dependent
 devide->divide
 devided->divided
 devider->divider
 deviders->dividers
 devides->divides
 deviding->dividing
+devision->division
 devistating->devastating
+devive->device
 devolopement->development
 devuce->device
+dezibel->decibel
 dezine->design
 diable->disable
 diabled->disabled
 diables->disables
 diablical->diabolical
 diabling->disabling
 diagnotic->diagnostic
+diagonale->diagonal
+diagramas->diagrams
+diamon->diamond
 diamons->diamonds
 diangose->diagnose
 dianostic->diagnostic
 diaplay->display
 diaplays->displays
+diappeares->disappearss
```

Comment:
disappearss should be disappears

Reply:
yikes, i thought i fixed that. Thanks for catching it.